WordPress/src/main/java/org/wordpress/android/ui/comments/CommentDetailFragment.java

```diff
@@ -1020,7 +1019,7 @@ private void setModerateButtonForStatus(CommentStatus status) {
             mBtnModerateTextView.setText(R.string.mnu_comment_approve);
         }
 
-        mBtnModerateIcon.setImageTintList(ColorStateList.valueOf(color));
+        ColorUtils.INSTANCE.setImageResourceWithTint(mBtnModerateIcon, R.drawable.ic_checkmark_white_24dp, color);
```

Comment:
Glad to see that with this change, one can now navigate the (java) code and notice the different icons the button can display. Previously, one of the cases was set vie the xml layout file. 👍